source/systems/hera_user_guide.rst

```diff
@@ -18,10 +18,21 @@ as well as its other environmental research areas.
 There are currently two major systems at NESCC:
 
 - Hera* A 760 Tflop Cray Compute Cluster high performance computing
-  system
+  system.
 - HPSS* A 50 Petabyte IBM/Oracle hierarchical storage management
   system.
 
+In 2025, two systems are scheduled for addition to NESCC:
+
+- Ursa, based on AMD 9654 and DDN Lustre, should be in place early in 2025.
+- Rhea, based on AMD Turin and DDN Lustre & VAST File Systems, is expected in
```

Comment:
```suggestion
- Ursa, based on AMD 9654 with DDN Lustre, should be in place early in 2025.
- Rhea, based on AMD Turin with DDN Lustre & VAST File Systems, is expected in
```

Reply:
After speaking with Raghu, it may be more proper to say "AMD with DDN Lustre". Otherwise, the changes look good to me, and I'll approve after Elizabeth reviews this suggestion.